<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_libro_iva_compras</class>
 <widget class="QWidget" name="form_libro_iva_compras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libro IVA Compras</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(3, 3, 3);
background-color: rgb(44, 43, 58);
font: 10pt &quot;Liberation Sans&quot;;
color: rgb(252, 252, 252);
selection-background-color: rgb(155, 155, 185);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>CUIT / CUIL: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_cuit">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Razón Social: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_razon_social">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btn_buscar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64, 63, 78);</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget{
background-color: rgb(123, 121, 143);
}
QLineEdit {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QComboBox{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Libro IVA Compras </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tw_compras">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>175</number>
         </attribute>
         <column>
          <property name="text">
           <string>Fecha</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>N° Comprobante</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Proveedor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Neto</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Resultado: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_resultado_uno">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_resultado_dos">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_eliminar">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(64, 63, 78);</string>
           </property>
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="btn_modificar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(64, 63, 78);</string>
           </property>
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="btn_nuevo">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(64, 63, 78);</string>
           </property>
           <property name="text">
            <string>Nuevo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lne_cuit</tabstop>
  <tabstop>lne_razon_social</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_nuevo</tabstop>
  <tabstop>btn_modificar</tabstop>
  <tabstop>tw_compras</tabstop>
  <tabstop>lne_resultado_uno</tabstop>
  <tabstop>lne_resultado_dos</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
